bookkeeper-server/src/test/java/org/apache/bookkeeper/client/api/BookKeeperApiTest.java

```diff
@@ -203,6 +203,12 @@ public void testOpenLedgerRead() throws Exception {
             assertEquals(2, result(reader.tryReadLastAddConfirmed()).intValue());
             checkEntries(result(reader.read(0, reader.getLastAddConfirmed())), data);
             checkEntries(result(reader.readUnconfirmed(0, reader.getLastAddConfirmed())), data);
+
+            // test readLastAddConfirmedAndEntry
+            LastConfirmedAndEntry lastConfirmedAndEntry =
+                result(reader.readLastAddConfirmedAndEntry(0, 999, false));
```

Comment:
We have a 999 timeout less then 1 second. This test could be flaky.
Maybe we can do a loop until the condition is verified

Reply:
@eolivelli Thanks.  
This is only for test the interface works, There are other tests for the functionality of asyncReadLastConfirmedAndEntry().
Here it read an old entry, which has already been there, so ideally it need no wait time.